Automattic/woocommerce-payments
# fix: drag handles display

Fixes #

#### Changes proposed in this Pull Request









I noticed that on `develop` the handles are still displayed when tabbing.
For accessibility reasons, they should not be displayed/tabbable.

This is how it's behaving in `develop`:
![2021-05-05 11 47 15](https://user-images.githubusercontent.com/273592/117178468-aed82580-ad97-11eb-9ac6-37650db981bd.gif)


This is how it should behave:
![2021-05-05 11 45 42](https://user-images.githubusercontent.com/273592/117178365-910ac080-ad97-11eb-99f9-6c288a878fd8.gif)


#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- Clicking the "delete" button on one of the enabled payment methods
- The drag handle should not be clickable/tabbable

-------------------
